docs/receiving/source-ips.mdx

```diff
@@ -0,0 +1,13 @@
+---
+title: Source IP Addresses
+---
+
+In case your webhook receiving endpoint is behind a firewall or a NAT, you may need to allow traffic from the Svix IP addresses.
```

Comment:
`or a NAT` -> `or NAT` and `from the Svix IP addresses.` -> `from Svix's IP addresses.`